### PR TITLE
docs: add usage and development instructions to README

### PR DESCRIPTION
**Pull request by greatislander**
This PR adds a full readme with usage and development instructions.

### CODE REVIEW

**Comment by greatislander** [comment]
@the-t-in-rtf in case you want to take a look too—I can't request your review for some reason.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,80 @@
 # stylelint-config-fluid
+
+[![License](https://badgen.net/github/license/fluid-project/stylelint-config-fluid/)](https://github.com/fluid-project/stylelint-config-fluid/blob/main/LICENSE.md)
```
Comment (jobara) [inline README.md:3]: I think it would help to add the license to the alt text; for example "License: BSD-3-Clause".

File: README.md [hunk 2]
```diff
@@ -1,2 +1,80 @@
...
+
+See the [stylelint user guide](https://stylelint.io/user-guide/configure#extends) for more options.
+
+## Developing
+
+See the [stylelint user guide](https://stylelint.io/user-guide/configure) for full details on configuration techniques.
+
+### Modifying configuration
+
+To modify the stylelint rules provided by this module, update the configuration in the [].stylelintrc.json](.stylelintrc.json) file. See the [stylelint user guide](https://stylelint.io/user-guide/configure) for configuration options.
```
Comment (jobara) [inline README.md:49]: There appears to be something broken in the markdown syntax, I think you meant the following.
```suggestion
To modify the stylelint rules provided by this module, update the configuration in the [.stylelintrc.json](.stylelintrc.json) file. See the [stylelint user guide](https://stylelint.io/user-guide/configure) for configuration options.
```

**Comment by jobara** [comment]
merged at e850ce2e7242b2bbd7feadec6f41cd76e0d394a2

**Pull request merged**